firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenGenerationConfig.kt

```diff
@@ -39,20 +39,49 @@ public class ImagenGenerationConfig(
    *
    * This is mainly intended for Java interop. For Kotlin, use [imagenGenerationConfig] for a more
    * idiomatic experience.
-   *
-   * @property negativePrompt See [ImagenGenerationConfig.negativePrompt].
-   * @property numberOfImages See [ImagenGenerationConfig.numberOfImages].
-   * @property aspectRatio See [ImagenGenerationConfig.aspectRatio].
-   * @property imageFormat See [ImagenGenerationConfig.imageFormat]
-   * @property addWatermark See [ImagenGenerationConfig.addWatermark]
-   * @see [imagenGenerationConfig]
    */
   public class Builder {
-    @JvmField public var negativePrompt: String? = null
-    @JvmField public var numberOfImages: Int? = 1
-    @JvmField public var aspectRatio: ImagenAspectRatio? = null
-    @JvmField public var imageFormat: ImagenImageFormat? = null
-    @JvmField public var addWatermark: Boolean? = null
+    private var negativePrompt: String? = null
```

Comment:
by making these private, are you breaking the Kotlin builder pattern?

Reply:
yeah im pretty sure it will. if this was in Java it wouldn't because Kotlin would translate it, but since it's written in Kotlin it _does_ break it.

Reply:
Good catch! Fixed